js/viewer.js

```diff
@@ -292,21 +333,32 @@ function receiveMenuContent(menuContent) {
 
     populateMenu(menu);
     redirectUrls(menu);
-
     updateSelection();
 }
 
+function updateMenuScrollbar() {
+    var e = document.documentElement;
+    var t = document.documentElement.scrollTop || document.body.scrollTop;
+    var p = e.scrollHeight - t - e.clientHeight;
+    // console.log("height = " + e.scrollHeight + " scrollTop = " + t + " clientHeight = " + e.clientHeight);
+    if (p < 244)
```

Comment:
There are several hard coded integers in this function, and it seems important to document their origin.